server/store/api.go

```diff
@@ -25,6 +25,8 @@ type KVStore interface {
 	Load(key string) ([]byte, error)
 	Store(key string, data []byte) error
 	Delete(key string) error
+	setJSON(s KVStore, key string, v interface{})
+	loadJSON(s KVStore, key string, v interface{})
```

Comment:
What value does a sealed interface provide here? https://blog.chewxy.com/2018/03/18/golang-interfaces/

Reply:
@hanzei  bit confused about this comment [here](https://github.com/mattermost/mattermost-plugin-splunk/pull/49#discussion_r744607216)  
i thought you wanted this function to be a part of KVStore interface
can you help me to understand this

Reply:
I've asked you to make it a method of `store`, but that doesn't mean it needs to be in the interface.